app/bundles/EmailBundle/EventListener/BuilderSubscriber.php

```diff
@@ -322,13 +322,9 @@ public function convertUrlsToTokens(EmailSendEvent $event): void
             return;
         }
 
-        // If a link shortener URL is provided, and shortening is enabled for e-mail, use it for shortening URLs in e-mails
-        $shorten_url     = (null != $this->coreParametersHelper->get('link_shortener_url', null) ? true : false);
-        $shorten_enabled = $this->coreParametersHelper->get('link_shortener_enable_email', false);
-        $shorten         = $shorten_url && $shorten_enabled;
-
-        $email   = $event->getEmail();
-        $emailId = ($email) ? $email->getId() : null;
+        $shortenEnabled = $this->coreParametersHelper->get('shortener_email_enable', false);
+        $email          = $event->getEmail();
+        $emailId        = ($email) ? $email->getId() : null;
```

Comment:
I know this line has been there before, but it looks weird when the following line is asking `if (!$email instanceof Email) {` If `$email` wasn't type of `Email` entity then calling  `$email->getId()` would fail, right?